### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    name: pyquil - Build and test module
+    strategy:
+      matrix:
+        os: ['ubuntu-20.04', 'macos-11', 'windows-2019']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: '0'
+    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
```
Comment (cqc-alec) [inline .github/workflows/build_and_test.yml:28]: On Linux, should pull docker images here (see `build_and_test.yml` in pytket-extensions repo).

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (cqc-melf) [reply .github/workflows/build_and_test.yml:28]: oh, I messed that. Thank you for spotting!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,83 @@
+# Pytket Extensions
+
+This repository contains the pytket-pyquil extension, using Quantinuum's
+[pytket](https://cqcl.github.io/tket/pytket/api/index.html) quantum SDK.
+The other pytket extensions can be found [here](https://github.com/CQCL/pytket-extensions)
```
[the remark this answers is not in the document]
Comment (cqc-melf) [reply README.md]: done

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (cqc-melf) [reply .github/workflows/build_and_test.yml:28]: I have added them now.

**Pull request merged**